cdap-docs/faq/source/cdap-startup.rst

```diff
@@ -0,0 +1,226 @@
+parcel, *Activation*. There are 4 steps to installing a parcel:
+
+- **Adding the repository** to the list of repositories searched by Cloudera Manager
+- **Downloading** the parcel to the Cloudera Manager server
+- **Distributing** the parcel to all the servers in the cluster
+- **Activating** the parcel
+
+Start by clicking on the parcel icon (near the top-left corner of Cloudera Manager; looks
+like a gift-wrapped box), and ensure that the CDAP parcel is listed as *Active*.
+
+
+I've followed the install instructions, yet CDAP does not start and fails verification. What next?
+--------------------------------------------------------------------------------------------------
+If you have followed :ref:`the installation instructions <install>`, and CDAP either did not pass the 
+:ref:`verification step <configuration-verification>` or did not startup, check:
+
+- Look in the CDAP logs for error messages (located in ``$CDAP_HOME/logs``)
+- If you see an error such as::
+
+    ERROR [main:c.c.c.StandaloneMain@268] - Failed to start Standalone CDAP
+    java.lang.NoSuchMethodError: 
+    co.cask.cdap.UserInterfaceService.getServiceName()Ljava/lang/String
+
+  then you have probably downloaded the standalone version of CDAP, which is not intended
+  to be run on Hadoop clusters. Download the appropriate distributed packages (RPM or
+  Debian version) from http://cask.co/downloads.
+         
+- Check permissions of directories:
+
+  - The :ref:`CDAP HDFS User <configuration-options>` (by default, ``yarn``) owns the HDFS directory (by default,  ``/cdap``).
+  - The :ref:`Kafka Log directory <configuration-options>` (by default, ``/data/cdap/kafka-logs``), must be writable by the default CDAP user.
+  - The :ref:`temp directories <configuration-tmp-files>` (by default, ``/tmp`` and ``/tmp/kafka-logs``) utilized by CDAP must be writable by the default CDAP user.
+
+.. - Check :ref:`configuration troubleshooting <configuration-troubleshooting>` suggestions
+   
+    
+.. CDAP UI shows a blank screen
+.. ----------------------------
+.. TBC.
+
+
+The CDAP UI is showing a message "namespace cannot be found".
+-------------------------------------------------------------
+This is indicative that the UI cannot connect to the CDAP system service containers running in YARN.
+
+- First, check if the CDAP Master service container shows as RUNNING in the YARN ResourceManager UI.
+
+- If this doesn't resolve the issue, then it means the CDAP system services were unable to launch.
+  Ensure :ref:`YARN has enough spare memory and vcore capacity <faq-installation-startup-memory-core-requirements>`.  
+  CDAP attempts to launch between 8 and 11 containers, depending on the configuration. Check
+  the master container (Application Master) logs to see if it was able to launch all containers.
+
+- If it was able to launch all containers, then you may need to further check the launched container logs for any errors.
```

Comment:
We should point to the yarn-site.xml configuration which sets the container log directory, so the user knows where to look. If we don't, it'll end up being a support question for us.